Kernel/FileSystem/SysFS/Subsystems/Kernel/RequestPanic.cpp

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) 2023, Liav A. <liavalb@hotmail.co.il>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <Kernel/FileSystem/SysFS/Subsystems/Kernel/RequestPanic.h>
+#include <Kernel/Library/Panic.h>
+
+namespace Kernel {
+
+UNMAP_AFTER_INIT NonnullRefPtr<SysFSSystemRequestPanic> SysFSSystemRequestPanic::must_create(SysFSDirectory const& parent_directory)
+{
+    return adopt_ref_if_nonnull(new (nothrow) SysFSSystemRequestPanic(parent_directory)).release_nonnull();
+}
+
+ErrorOr<size_t> SysFSSystemRequestPanic::read_bytes(off_t, size_t, UserOrKernelBuffer&, OpenFileDescription*) const
+{
+    return Error::from_errno(ENOTSUP);
+}
+
+ErrorOr<size_t> SysFSSystemRequestPanic::write_bytes(off_t, size_t, UserOrKernelBuffer const&, OpenFileDescription*)
+{
+    PANIC("SysFSSystemRequestPanic::write_bytes");
```

Comment:
Maybe I should check if the process is within a jail? otherwise jailed process could execute code (if it has root privileges) and crash the OS.

Reply:
I thought SysFS was completely hidden in jails anyways

Reply:
it's not hidden - it's just not writable in most cases and isn't readable in some cases too.